Update to v20.11 to align with SPDK v21.01

tanabarr opened the pull request:
Update DPDK to provide dependency for SPDK in 
https://github.com/daos-stack/spdk/pull/14

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
Update to v20.11 to align with SPDK v21.01

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

brianjmurrell commented on dpdk.spec:
```diff
@@ -313,6 +309,9 @@ sed -i -e 's:-%{machine_tmpl}-:-%{machine}-:g' %{buildroot}/%{_sysconfdir}/profi
 %endif
 
 %changelog
+* Thu Jan 29 2021 Tom Nabarro <tom.nabarro@intel.com> - 0:20.11-1
```
```suggestion
* Thu Jan 28 2021 Tom Nabarro <tom.nabarro@intel.com> - 0:20.11-1
```

tanabarr replied:
it was 29th in my time zone :-)

brianjmurrell commented:
The spdk team usually have some number of patches on their dpdk.  Is this not the case this time around?

tanabarr committed:
Update dpdk.spec

Co-authored-by: Brian J. Murrell <brian.murrell@intel.com>

tanabarr committed:
add patch to specify exact commit used for spdk v21.01-rc1 release

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
add patch target to makefile

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
fix packaging

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
update packaging

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
add patch file

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
add back newline

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
update packaging

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
try to move to meson/ninja for build and install in specfile

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
specify meson src & build dirs

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
remove unused packages and remove unpackaged files

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
appease linter

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
add libisa-l-devel build requirement

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
disable compress/isal driver in build

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
remove isal build dep from spec

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
fix doc dir for sles in specfile

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
backport dpdk deb pkg from hirsute ubuntu release

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
remove unused patches

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
remove ubuntu build from Jenkinsfile, to be addressed in separate PR

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

tanabarr committed:
don't remove wildcard links

Signed-off-by: Tom Nabarro <tom.nabarro@intel.com>

brianjmurrell committed:
Updates to re-establish pre-existing sub-packages

Skip-checkpatch: true
Skip-build: true

Quick-build: true

Skip-unit-tests: true

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>
Skip-test: true

brianjmurrell committed:
Merge branch 'tanabarr/update-20_11' of github.com:daos-stack/dpdk into tanabarr/update-20_11

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>

brianjmurrell committed:
Updates

Signed-off-by: Brian J. Murrell <brian.murrell@intel.com>
